envVariables.js

```diff
@@ -61,127 +61,165 @@ let enviromentVariables = {
+  //   "optional" : true,
+  //   "requiredIf" : {
+  //     "key": "CLOUD_STORAGE",
+  //     "operator": "EQUALS",
+  //     "value" : "OCI"
+  //   }
+  // }, 
+  // "OCI_BUCKET_REGION" : {
+  //   "message" : "Required Oracle bucket region",
+  //   "optional" : true,
+  //   "requiredIf" : {
+  //     "key": "CLOUD_STORAGE",
+  //     "operator": "EQUALS",
+  //     "value" : "OCI"
+  //   }
+  // }, 
+  // "OCI_BUCKET_ENDPOINT" : {
+  //   "message" : "Required Oracle bucket endpoint",
+  //   "optional" : true,
+  //   "requiredIf" : {
+  //     "key": "CLOUD_STORAGE",
+  //     "operator": "EQUALS",
+  //     "value" : "OCI"
+  //   }
+  // },
+
+
+
+
+  "CLOUD_STORAGE_PROVIDER" : {
+    "message" : "Require cloud storage provider",
     "optional" : false
   },
-  "GCP_PATH" : {
-    "message" : "Required Gcp path",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "GC"
-    }
+  "CLOUD_STORAGE_BUCKETNAME" : {
+    "message" : "Require client storage bucket name",
+    "optional" : false
   },
-  "DEFAULT_GCP_BUCKET_NAME" : {
-    "message" : "Required Gcp bucket name",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "GC"
-    }
+  "CLOUD_STORAGE_SECRET" : {
+    "message" : "Require client storage provider identity",
+    "optional" : false
   },
-  "AZURE_ACCOUNT_NAME" : {
-    "message" : "Required Azure Account name",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AZURE"
-    }
+  "CLOUD_STORAGE_REGION" : {
+    "message" : "Require client region",
+    "optional" : false
   },
-  "AZURE_ACCOUNT_KEY" : {
-    "message" : "Required Azure Account key",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AZURE"
-    }
+  "CLOUD_ENDPOINT" : {
+    "message" : "Require client endpoint",
+    "optional" : false
   },
-  "AWS_ACCESS_KEY_ID" : {
-    "message" : "Required Aws access key id",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AWS"
-    }
-  }, 
-  "AWS_SECRET_ACCESS_KEY" : {
-    "message" : "Required Aws secret access key",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AWS"
-    }
-  }, 
-  "DEFAULT_AWS_BUCKET_NAME" : {
-    "message" : "Required Aws bucket name",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AWS"
-    }
-  }, 
-  "AWS_BUCKET_REGION" : {
-    "message" : "Required Aws bucket region",
-    "optional" : true,
-    "requiredIf" : {
-      "key": "CLOUD_STORAGE",
-      "operator": "EQUALS",
-      "value" : "AWS"
-    }
+  "CLOUD_STORAGE_ACCOUNTNAME" : {
```

Comment:
provider: process.env.CLOUD_STORAGE_PROVIDER,
      identity: process.env.CLOUD_STORAGE_ACCOUNTNAME,
      credential: process.env.CLOUD_STORAGE_SECRET,
      publicContainer: process.env.CLOUD_STORAGE_BUCKETNAME,
      @Prajwal17Tunerlabs  Above mentioned are mandatory. remove if anything else is specified